index.md

```diff
@@ -21,6 +21,15 @@ title: Distributed SQL query engine for big data
   </div>
 </div>
 
+<div class="trino-birthday-banner">
+  <div class="trino-birthday-content">
+    <img src="/assets/images/trino-summit-logo.png" class="trino-birthday-logo" />
```

Comment:
this needs to be changed to birthday image..

Reply:
Currently waiting on those